src/test/java/com/wikia/webdriver/elements/communities/desktop/components/categories/CategoryLayoutSelector.java

```diff
@@ -0,0 +1,59 @@
+package com.wikia.webdriver.elements.communities.desktop.components.categories;
+
+import com.wikia.webdriver.elements.communities.desktop.pages.categories.CategoryExhibitionPage;
+import com.wikia.webdriver.elements.communities.desktop.pages.categories.DynamicCategoryPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * This component lets user choose category page layout, it's available only for logged in users
+ */
+public class CategoryLayoutSelector extends BasePageObject {
+
+  @FindBy(css = "li[data-category-layout=mediawiki]")
+  private WebElement classicViewButton;
+  @FindBy(css = "li[data-category-layout=category-exhibition]")
+  private WebElement categoryExhibitionButton;
+  @FindBy(css = "li[data-category-layout=category-page3]")
+  private WebElement dynamicViewButton;
+  @FindBy(css = ".category-layout-selector")
+  private WebElement ViewContainer;
+
+  public static CategoryLayoutSelector getComponent() {
+    //TODO: Chack if user is logged in, if not throw exception
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)